content/training/R_Basics/Advanced_Functions.Rmd

```diff
@@ -19,17 +19,21 @@ linelist_cleaned <- gen_data("AJS")
 ```
 
 
-### Factors and the forcats() package  
+### Factors and the *forcats* package  
 
 #### Factors
 
 Factors are a special variable class in which data are stored internally as integers, but each integer *level* has a corresponding character value *label* that is used when the data are displayed. This can be partularly helpful when running statistical models which expect numeric (e.g. 0/1) inputs.  
 
-*Factors are similar to labeled values of categorical variables in Stata.*  
+{{% notice tip %}}
 
-You can read more about factors [here](https://www.stat.berkeley.edu/~s133/factors.html).  
+Factors are similar to labeled values of categorical variables in Stata.*  
```

Comment:
Is the asterisk at the end of this line an accident or on purpose?

Reply:
Looks like the starting one got dropped somehow.
```suggestion
*Factors are similar to labeled values of categorical variables in Stata.*  
```

Reply:
it got dropped because I was trying to remove the formatting and forgot to remove the closing asterix